examples/rl/platoon/train/utils.py

```diff
@@ -0,0 +1,15 @@
+class ObjDict(dict):
+    def __getattr__(self, name):
+        if name in self:
+            return self[name]
+        else:
+            raise AttributeError("No such attribute: " + name)
+
+    def __setattr__(self, name, value):
+        self[name] = value
+
+    def __delattr__(self, name):
+        if name in self:
+            del self[name]
+        else:
+            raise AttributeError("No such attribute: " + name)
```

Comment:
Yes, it is fun. `ObjDict` helps to access a key in a dictionary using `.` (similar to accessing attributes in a dataclass), instead of using string `dict["key"]`. I felt the former leads to nicer code and thus used it in the training script.